src/CoreShop/Bundle/CoreBundle/Migrations/Version20180523085857.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace CoreShop\Bundle\CoreBundle\Migrations;
+
+use CoreShop\Component\Pimcore\DataObject\ClassUpdate;
+use Doctrine\DBAL\Schema\Schema;
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+class Version20180523085857 extends AbstractPimcoreMigration implements ContainerAwareInterface
+{
+    use ContainerAwareTrait;
+
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
```

Comment:
That's a bc break right? What about just adding a upgrade note? As long no link generator has been defined the default pathgenerator gets triggered => no bc break

Reply:
Well, do you mean cause of the slug? Otherwise: No. The DataObjectLinkGenerator does the same as the old version did except for the slug -> which IMO is a BC break. We could remove the migration, would be fine with me.

Reply:
just checked it. the migration is fine since you need to change the twig extension from `path` to `coreshop_path` anyway. If you keep the `path` the url isn't changing so there is no bc break after all. 👍

Reply:
Good.

Reply:
BUT: What if you didn't override the template?

Reply:
you're out of luck.